Bottom: 23f04184d4099129f031f7fd3fd76a715ed588e1
Top:    ac73ff0db6ce5b9611b2e32509949f660411c62c
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-30 13:51:31 +0100

Refresh of rats-c-tidy-up-findpad

---

diff --git a/src/rats.c b/src/rats.c
index 24a3007..a6e526c 100644
--- a/src/rats.c
+++ b/src/rats.c
@@ -123,7 +123,7 @@ static bool
 FindPad (char *ElementName, char *PinNum, ConnectionType * conn, bool Same)
 {
   ElementTypePtr element;
-  GList *i;
+  Cardinal i;
 
   if ((element = SearchElementByName (PCB->Data, ElementName)) == NULL)
     return false;
